src/plugins/home/server/services/new_instance_status.test.ts

```diff
@@ -0,0 +1,129 @@
+    soClient.find.mockResolvedValue({
+      page: 1,
+      per_page: 100,
+      total: 0,
+      saved_objects: [],
+    });
+    expect(await isNewInstance({ esClient, soClient })).toEqual(true);
+  });
+
+  it('returns false when there are any index patterns other than metrics-* or logs-*', async () => {
+    soClient.find.mockResolvedValue({
+      page: 1,
+      per_page: 100,
+      total: 1,
+      saved_objects: [
+        {
+          id: '1',
+          references: [],
+          type: 'index-pattern',
+          score: 99,
+          attributes: { title: 'my-pattern-*' },
+        },
+      ],
+    });
+    expect(await isNewInstance({ esClient, soClient })).toEqual(false);
+  });
+
+  describe('when only metrics-* and logs-* index patterns exist', () => {
+    beforeEach(() => {
+      soClient.find.mockResolvedValue({
+        page: 1,
+        per_page: 100,
+        total: 2,
+        saved_objects: [
+          {
+            id: '1',
+            references: [],
+            type: 'index-pattern',
+            score: 99,
+            attributes: { title: 'metrics-*' },
+          },
+          {
+            id: '2',
+            references: [],
+            type: 'index-pattern',
+            score: 99,
+            attributes: { title: 'logs-*' },
+          },
+        ],
+      });
+    });
+
+    it('calls /_cat/indices for the index patterns', async () => {
+      await isNewInstance({ esClient, soClient });
+      expect(esClient.asCurrentUser.cat.indices).toHaveBeenCalledWith({
+        index: 'logs-*,metrics-*',
+        format: 'json',
+      });
+    });
+
+    it('returns true if no logs or metrics indices exist', async () => {
+      esClient.asCurrentUser.cat.indices.mockReturnValue(
+        elasticsearchServiceMock.createSuccessTransportRequestPromise([])
+      );
+      expect(await isNewInstance({ esClient, soClient })).toEqual(true);
+    });
+
+    it('returns true if no logs or metrics indices contain data', async () => {
+      esClient.asCurrentUser.cat.indices.mockReturnValue(
+        elasticsearchServiceMock.createSuccessTransportRequestPromise([
+          { index: '.ds-metrics-foo', docsCount: '0' },
+        ])
+      );
+      expect(await isNewInstance({ esClient, soClient })).toEqual(true);
+    });
+
+    it('returns true if only metrics-elastic_agent index contains data', async () => {
+      esClient.asCurrentUser.cat.indices.mockReturnValue(
+        elasticsearchServiceMock.createSuccessTransportRequestPromise([
+          { index: '.ds-metrics-elastic_agent', docsCount: '100' },
+        ])
+      );
+      expect(await isNewInstance({ esClient, soClient })).toEqual(true);
+    });
+
+    it('returns true if only logs-elastic_agent index contains data', async () => {
+      esClient.asCurrentUser.cat.indices.mockReturnValue(
+        elasticsearchServiceMock.createSuccessTransportRequestPromise([
+          { index: '.ds-logs-elastic_agent', docsCount: '100' },
```

Comment:
Seems should be:

```suggestion
          { index: '.ds-logs-elastic_agent', 'docs.count': '100' },
```

And the same in other tests?